bazel/utils/exec_test.bzl

```diff
@@ -0,0 +1,59 @@
+load("@bazel_skylib//lib:shell.bzl", "shell")
+
+def _exec_test(ctx):
+    # A test rule must return an executable file, cannot reuse
+    # the executable returned by another rule.
+    template = """#!/bin/sh
+ARGS={argv}; ARGS+=("$@")
+exec {script} "$ARGS[@]"
```

Comment:
Are you sure this syntax works with `/bin/sh` ?  This looks like `/bin/bash` syntax.

Reply:
good catch, fixing it.